### ISSUE TITLE
Development of a dashboard

### ISSUE THREAD

**Issue by vinayaksh42** [Bot-Academia/Photon#3]
a dashboard like user interface with locally stored data.

**Comment by codetheorem** [comment]
We will make it a progressive web app.

**Comment by vinayaksh42** [comment]
@gauravchauhan2403 let's get started with this.

**Comment by gauravchauhan2403** [comment]
yes sure but can we first discuss about it's blueprint and also the color scheme

**Comment by codetheorem** [comment]
https://madewithreactjs.com/dashboards take reference from this.
